### ISSUE TITLE
TypeError: DeciLMForCausalLM.__init__() takes from 1 to 3 positional arguments but 4 were given

### ISSUE THREAD

**Issue by ggbetz** [vllm-project/vllm#2690]
Hi! Thx for the new release. I just realized that loading DeciLM doesn't seem to work anymore in `0.3.0`.

```python
from vllm import LLM

llm = LLM(model="Deci/DeciLM-7B", trust_remote_code=True)  # Name or path of your model
```
Throws:
```
[/usr/local/lib/python3.10/dist-packages/vllm/model_executor/model_loader.py](https://localhost:8080/#) in get_model(model_config, lora_config)
     70         with torch.device("cuda"):
     71             if getattr(model_class, "supports_lora", False):
---> 72                 model = model_class(model_config.hf_config, linear_method,
     73                                     lora_config)
     74             elif lora_config:

TypeError: DeciLMForCausalLM.__init__() takes from 1 to 3 positional arguments but 4 were given
```

This did work in `0.2.7`.

Tested (i) in colab and (ii) on recent NVIDIA docker image with A6000.

**Comment by s-jse** [comment]
I've run into the exact same issue as well.

**Comment by esmeetu** [comment]
Hi, @ggbetz @ggbetz . This issuce has been solved in v0.3.1.
